src/citrine/exceptions.py

```diff
@@ -84,6 +85,21 @@ class WorkflowNotReadyException(RetryableException):
     pass
 
 
+class PollingTimeoutError(RetryableException):
```

Comment:
Is this retryable?  Seems like if the polling times out the user should not simply repeat the request.

Reply:
I suppose one could retry the poll if they wanted to wait longer. I'm happy to make it a non-retryable exception.

Reply:
Done.